meganav.yaml

```diff
@@ -10,12 +706,6 @@ get_ubuntu:
                 - title: Download 22.04 LTS
                   url: /download/desktop
                   type: primary
-                - title: "21.10"
-                  url: /download
-                  type: secondary
-                - title: All
-                  url: /download
-                  type: secondary
             - title: Ubuntu flavours
               description: Variations of Ubuntu OS
               url: /download/flavours
```

Comment:
below, 'Ubuntu ready PC and Laptops' is missing the description 'Certified Hardware'